### ISSUE TITLE
Web interface edit host configuration and active hosting

### ISSUE THREAD

**Issue by seveibar** [NebulousLabs/Sia#193]
The web interface host configuration should be editable and the user should be able to active hosting.

**Comment by seveibar** [comment]
I don't have the API calls to populate the host config yet

**Comment by DavidVorick** [comment]
should be ready in a few hours

**Comment by seveibar** [comment]
The latest API change for host allows it to be set, but I still can't retrieve config values.

Maybe instead of `/host` (which is for setting the config), make `/host/config` and `/host/setconfig` where `/host/setconfig` is equivalent to the current `/host`.

**Comment by DavidVorick** [comment]
still getting there, =/ sorry it's taking a while.

Getting sidetracked by the signing bug.

**Comment by seveibar** [comment]
Ideally `/host/config` would return

```javascript
{
  "HostSettings": {
    "IPAddress": ":9988",
    "TotalStorage": 0,
    "MinFilesize": 0,
    "MaxFilesize": 0,
    "MinDuration": 0,
    "MaxDuration": 0,
    "MinChallengeWindow": 0,
    "MaxChallengeWindow": 0,
    "MinTolerance": 0,
    "Price": 0,
    "Burn": 0,
    "CoinAddress": [
      0,
      0,
      "...",
      0,
      0
    ],
    "SpendConditions": {
      "TimeLock": 0,
      "NumSignatures": 0,
      "PublicKeys": null
    },
    "FreezeIndex": 0
  }
}```

**Comment by DavidVorick** [comment]
fixed with #210
